pages/index.tsx

```diff
@@ -29,8 +32,8 @@ export default function Home({ claimData, loading }: HomeProps): ReactElement {
         <link href="https://fonts.googleapis.com/css?family=Source Sans Pro" rel="stylesheet" />
       </Head>
       <WorkInProgress />
-      <Header />
-      <Main loading={loading} />
+      <Header mobile={mobile} />
+      <Main loading={loading} mobile={mobile} />
```

Comment:
I think this would make a useful enough inline code comment? I feel like in general if it's useful enough to bother explaining code intent to a PR reviewer, it's likely to be useful enough to inline comment

Reply:
I actually disagree here! I was commenting just for the sake of 'how this PRs changes are all related', which I don't think is useful to have documented in the code longterm. It would also be documenting one components internal behavior in a different component, which is a surefire way to get outdated when things change.

That said I renamed the variable to `userArrivedFromUioMobile` like you suggested, which is much clearer to understand now that 'this component cares for some reason that the user came from mobile'